cmd/swapd/main_test.go

```diff
@@ -150,14 +151,14 @@ func Test_expandBootnodes(t *testing.T) {
 		"node7",
 		"node8",
 	}
-	require.EqualValues(t, expected, expandBootnodes(cliNodes))
```

Comment:
This whole test belongs in the package where the function it is testing is implemented.

Reply:
updated